can_tools/scrapers/official/ID/id_county.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import us
+
+from can_tools.scrapers.official.base import TableauDashboard
+from can_tools.scrapers import variables
+class IdahoCountyVaccine(TableauDashboard):
+    has_location = False
+    source = "https://coronavirus.idaho.gov/"
```

Comment:
I think we need to add `location_type = 'county'`

Reply:
Good catch